### DISCUSSION TITLE
Enhancement: Add example FastAPI app

### DISCUSSION THREAD

**Discussion by iron3oxide** [yezz123/ormdantic#46] [Ideas]
It would be great to have a simple example app in order to fully understand the benefit of using ormdantic and how to best use it. From my understanding ormdantic has been designed with FastAPI in mind, so a FastAPI app would probably be best. This could live fully in the docs or be a separate repo, whatever makes the most sense.

**Comment by yezz123** [comment]
Hello @iron3oxide 👋🏻 

I'm planning to create a repo ASAP for a simple demo on how we can use ormdantic with FastAPI & Flask also and provide different utilities for it!

I'm a bit run into other several works that's why but I will provide it or someone from the community gonna work on it and why not! 🚀 🕺🏻
